circle.yml

```diff
@@ -67,6 +67,11 @@ deployment:
     owner: palantir
     commands:
       - ./scripts/circle-ci/publish-github-page.sh
+  publish-snapshots:
+    branch: /.*/
```

Comment:
I initially had concerns with this (all branches) though can see why we might want it. Post offline discussion, it seems fine in that Artifactory should only keep the latest N snapshot builds and will clear them if needed to free up space.